### ISSUE TITLE
Flagging "incomplete submission" validation in CLI

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#1836]
### Standard

CDISC SDTM/SDTMIG

### Reference Rule ID(s)

N/A

### Proposed Rule YAML

```YAML
N/A.
One of the great advantages of CORE is that one does not need to wait until all datasets are complete, but that it can be used already when starting executing the first mappings, e.g. on a single dataset (-dp parameter in the CLI)
When doing so for LB only, the generated Excel reports an issue "EX is not present in study when study includes protocol-specified study treatment", although EX was not in the list submittted to the engine. Together with other "rule evaluation error" messages in the Excel, this is very confusing - the lay user will think that something has really gone wrong, which is of course not the case.
Therefore I would like to see a CLI parameter that takes care that rules that cannot be evaluated or executed as the dataset is not in the list of datasets submitted to the engine are either not executed at all, or that messages for these are excluded from the report.
```

### Attach any sample test data file(s)

N/A

### Expected output

N/A

**Comment by SFJohnson24** [comment]
Hi Jozef-- Users can curate the rules they are running using the -r or the -er command to make sure they are not running any rules that would throw an error for datasets they have not submitted to engine.  Users have access to cdisc-open-rules and can see all the rules by cloning the repository now and thus could exclude these presence check rules from their validation.  I also would push back that a user with an incomplete study would be surprised by an error that EX is not present when they dont submit one in the validation.   I guess I would ask for this feature--are there rules outside of domain presence checks for required domains where you would see this feature coming in handy?  I would think most of the rules would skip due to missing domains outside of these few rules that check for the missing required domains themselves

**Comment by JozefAerts** [comment]
Thanks Sam!
Don't underestimate the stupidness of the users ... ;-)
I think we can make something in our application on top of CORE to exclude such rules. when doing such "partial" validations. 
I still need to look how these "not present in the study" rules have been implemented in YAML/JSON, but I think we can find a way.
